fvtest/porttest/omrstrTest.cpp

```diff
@@ -1538,35 +1546,71 @@ TEST(PortStrTest, str_test_atoe_vsnprintf)
 
 	/* Neither min_width nor precision is specified for a null terminated input string*/
 	portTestEnv->log("\n\tTesting case 1\n");
-	rc |= printToBuffer(OMRPORTLIB, TRUE, buffer0, bufferLength0, expectedOutput1, "%s", nullTerminatedString);
+	rc |= printToBuffer(portLibrary, TRUE, buffer0, bufferLength0, expectedOutput1, strlen(expectedOutput1), "%s", nullTerminatedString);
```

Comment:
This code should continue to use `OMRPORTLIB` (I don't think `portLibrary` is even meaningful here).